src/vivarium_public_health/risks/distributions.py

```diff
@@ -76,31 +79,43 @@ def initialization_requirements(self) -> Dict[str, List[str]]:
     def __init__(self, risk, weights, mean, sd):
         super().__init__()
         self.risk = EntityString(risk)
-        self._weights, self._parameters = self.get_parameters(weights, mean, sd)
+        self.weights = weights
```

Comment:
In these distribution classes, we moved get_parameters out of the constructor to the setup method to have access to the builder. In turn, saving the other arguments from the constructor in order to be able to access them later.